sycl/doc/extensions/USM/USM.adoc

```diff
@@ -299,26 +331,36 @@ Parameters::
   * `size_t count` - number of bytes to copy
 Return value:: Returns an event representing the copy operation.
 
-===== memset
+===== memfill
 [source,cpp]
 ----
 class handler {
  ...
  public:
   ...
-  void memset(void* ptr, int value, size_t count);
+  (1)
+  void memfill(void* ptr, const void* pattern, size_t pattern_size, size_t count);
+  (2)
+  template <typename T>
+  void memfill(void* ptr, const T* pattern, size_t count)
 };
 
 class queue {
  ...
  public:
   ...
-  event memset(void* ptr, int value, size_t count);
+  (1)
+  event memfill(void* ptr, const void* pattern, size_t pattern_size, size_t count);
```

Comment:
Do we need a description of what `memfill` does, since there's not a C++ equivalent in `std::`?  For example, it might be useful to say that `ptr` is filled by copying the underlying bytes of `pattern`.

Reply:
Yep.

Reply:
Should this function return a valid `event` if `count == 0`? What about `pattern_size == 0`?